models/token.js

```diff
@@ -17,7 +17,7 @@ const MODEL = {
 
     hash: Joi
         .string()
-        .base64()
+        .regex(/[a-zA-Z0-9_-]+~/)
```

Comment:
So now you have to know that the characters in this `regex` are dependent on an implementation in a library that's not referenced anywhere in this file. It'll just suck for the next developer that has to debug this.